pkg/flag/db_flags_test.go

```diff
@@ -17,57 +17,71 @@ func TestDBFlagGroup_ToOptions(t *testing.T) {
 		SkipDBUpdate     bool
 		DownloadDBOnly   bool
 		Light            bool
-		DBRepository     string
-		JavaDBRepository string
+		DBRepository     []string
+		JavaDBRepository []string
 	}
 	tests := []struct {
-		name      string
-		fields    fields
-		want      flag.DBOptions
-		wantLogs  []string
-		assertion require.ErrorAssertionFunc
+		name     string
+		fields   fields
+		want     flag.DBOptions
+		wantLogs []string
+		wantErr  string
 	}{
 		{
 			name: "happy",
 			fields: fields{
 				SkipDBUpdate:     true,
 				DownloadDBOnly:   false,
-				DBRepository:     "ghcr.io/aquasecurity/trivy-db",
-				JavaDBRepository: "ghcr.io/aquasecurity/trivy-java-db",
+				DBRepository:     []string{"ghcr.io/aquasecurity/trivy-db"},
+				JavaDBRepository: []string{"ghcr.io/aquasecurity/trivy-java-db"},
 			},
 			want: flag.DBOptions{
-				SkipDBUpdate:     true,
-				DownloadDBOnly:   false,
-				DBRepository:     name.Tag{}, // All fields are unexported
-				JavaDBRepository: name.Tag{}, // All fields are unexported
+				SkipDBUpdate:       true,
+				DownloadDBOnly:     false,
+				DBRepositories:     []name.Reference{name.Tag{}}, // All fields are unexported
+				JavaDBRepositories: []name.Reference{name.Tag{}}, // All fields are unexported
+			},
+			wantLogs: []string{
+				`Adding schema version to the DB repository for backward compatibility	repository="ghcr.io/aquasecurity/trivy-db:2"`,
+				`Adding schema version to the DB repository for backward compatibility	repository="ghcr.io/aquasecurity/trivy-java-db:1"`,
 			},
-			assertion: require.NoError,
 		},
 		{
 			name: "sad",
 			fields: fields{
 				SkipDBUpdate:   true,
 				DownloadDBOnly: true,
 			},
-			assertion: func(t require.TestingT, err error, msgs ...any) {
-				require.ErrorContains(t, err, "--skip-db-update and --download-db-only options can not be specified both")
-			},
+			wantErr: "--skip-db-update and --download-db-only options can not be specified both",
 		},
 		{
 			name: "invalid repo",
 			fields: fields{
 				SkipDBUpdate:   true,
 				DownloadDBOnly: false,
-				DBRepository:   "foo:bar:baz",
+				DBRepository:   []string{"foo:bar:baz"},
 			},
-			assertion: func(t require.TestingT, err error, msgs ...any) {
-				require.ErrorContains(t, err, "invalid db repository")
+			wantErr: "invalid DB repository",
+		},
+		{
+			name: "multiple repos",
+			fields: fields{
+				SkipDBUpdate:     true,
+				DownloadDBOnly:   false,
+				DBRepository:     []string{"ghcr.io/aquasecurity/trivy-db:2", "gallery.ecr.aws/aquasecurity/trivy-db:2"},
```

Comment:
The 2nd default repository is wrong, it should be "public.ecr.aws/aquasecurity/trivy-db:2"